presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8336,6 +8336,19 @@ public void testInComplexTypes()
         assertQuery(query.toString(), "SELECT NULL");
     }
 
+    @Test
+    public void testRowExpressionInterpreterStackOverflow()
+    {
+        StringBuilder stringBuilder = new StringBuilder("SELECT  CASE");
+        // Generate 100 cases.
```

Comment:
isn't it 500 cases?

Reply:
Removed the comment to follow the "no comments" style :)